### PR TITLE
The first letter should be capitalized

### PR DESCRIPTION
**Pull request by yu8833**

### CODE REVIEW

File: vendor/src/github.com/coreos/etcd/raft/design.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ A progress changes from `probe` to `snapshot` when the follower falls very far b
 
 ### Flow Control
 
-1. limit the max size of message sent per message. Max should be configurable.
+1. Limit the max size of message sent per message. Max should be configurable.
 Lower the cost at probing state as we limit the size per message; lower the penalty when aggressively decreased to a too low `next`
 
-2. limit the # of in flight messages < N when in `replicate` state. N should be configurable. Most implementation will have a sending buffer on top of its actual network transport layer (not blocking raft node). We want to make sure raft does not overflow that buffer, which can cause message dropping and triggering a bunch of unnecessary resending repeatedly. 
+2. Limit the # of in flight messages < N when in `replicate` state. N should be configurable. Most implementation will have a sending buffer on top of its actual network transport layer (not blocking raft node). We want to make sure raft does not overflow that buffer, which can cause message dropping and triggering a bunch of unnecessary resending repeatedly.
```
Comment (thaJeztah) [inline vendor/src/github.com/coreos/etcd/raft/design.md]: This document is vendored from an external repository, so should not be changed here, but upstream; can you revert this change, and open a pull request to the upstream repository instead? This file can be found upstream in https://github.com/coreos/etcd/blob/master/raft/design.md

File: vendor/src/github.com/docker/libnetwork/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Please refer to the [design](docs/design.md) for more information.
 
 #### Using libnetwork
 
-There are many networking solutions available to suit a broad range of use-cases. libnetwork uses a driver / plugin model to support all of these solutions while abstracting the complexity of the driver implementations by exposing a simple and consistent Network Model to users.
+There are many networking solutions available to suit a broad range of use-cases. Libnetwork uses a driver / plugin model to support all of these solutions while abstracting the complexity of the driver implementations by exposing a simple and consistent Network Model to users.
```
Comment (thaJeztah) [inline vendor/src/github.com/docker/libnetwork/README.md]: Likewise; this document is vendored from an external repository, so should not be changed here, but upstream; can you revert this change, and open a pull request to the upstream repository instead? This file can be found upstream in https://github.com/docker/libnetwork/blob/master/README.md

File: vendor/src/github.com/docker/notary/README.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ received content.
...
-* **Freshness Guarantees**: Replay attacks are a common problem in designing secure systems, where previously valid payloads are replayed to trick another system. The same problem exists in the software update systems, where old signed can be presented as the most recent. notary makes use of timestamping on publishing so that consumers can know that they are receiving the most up to date content. This is particularly important when dealing with software update where old vulnerable versions could be used to attack users.
+* **Freshness Guarantees**: Replay attacks are a common problem in designing secure systems, where previously valid payloads are replayed to trick another system. The same problem exists in the software update systems, where old signed can be presented as the most recent. Notary makes use of timestamping on publishing so that consumers can know that they are receiving the most up to date content. This is particularly important when dealing with software update where old vulnerable versions could be used to attack users.
```
Comment (thaJeztah) [inline vendor/src/github.com/docker/notary/README.md]: Same here; This file can be found upstream in https://github.com/docker/notary/blob/master/README.md

**Comment by thaJeztah** [comment]
Thanks @YuPengZTE, looks like you made some modifications to vendored files; can you revert those changes and update this PR?

**Comment by thaJeztah** [comment]
closing this, because there's a new PR opened here https://github.com/docker/docker/pull/25613

**Comment by YuPengZTE** [comment]
Thanks @thaJeztah, this PR is closed, and #25613 is sloving the same problem.
